lib/processors/ide/vscode_launch_processor.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter_flavorizr/parser/models/flavors/flavor.dart';
+import 'package:flutter_flavorizr/processors/commons/copy_folder_processor.dart';
+import 'package:flutter_flavorizr/processors/commons/queue_processor.dart';
+import 'package:flutter_flavorizr/processors/commons/replace_string_processor.dart';
+import 'package:flutter_flavorizr/processors/commons/runtime_file_string_processor.dart';
+
+class VSCodeLaunchProcessor extends QueueProcessor {
+  VSCodeLaunchProcessor(
+      String source, String destination, Map<String, Flavor> flavors)
```

Comment:
You can pass directly the flavors.keys from outside

Reply:
Good catch.